client/src/app/site/motions/motions.constants.ts

```diff
@@ -68,31 +68,6 @@ export const verboseChangeRecoMode = {
     modified_final_version: 'Final print template'
 };
 
-/**
- * Enum to define different types of notifications.
- */
-export enum MotionEditNotificationType {
```

Comment:
You deleted this enum - but where is the new one?

Reply:
I have changed it to `EditNotificationType`

Reply:
Ah I oversaw these changes..